.github/workflows/kind-cluster-custom-resources.yaml

```diff
@@ -0,0 +1,180 @@
+
+      # will use the latest release available for ko
+      - uses: ko-build/setup-ko@3aebd0597dc1e9d1a26bcfdb7cbeb19c131d3037 # v0.7
+
+      - uses: imranismail/setup-kustomize@2ba527d4d055ab63514ba50a99456fc35684947f # v2.1.0
+
+      - name: Install yq
+        uses: mikefarah/yq@bbdd97482f2d439126582a59689eb1c855944955 # v4.44.3
+
+      - name: Setup mirror
+        uses: chainguard-dev/actions/setup-mirror@main
+        with:
+          mirror: mirror.gcr.io
+
+      - uses: sigstore/cosign-installer@dc72c7d5c4d10cd6bcb8cf6e3fd625a9e5e537da # v2
+
+      - name: Install cluster + sigstore
+        uses: sigstore/scaffolding/actions/setup@main
+        env:
+          INSTALL_TSA: true
+        with:
+          k8s-version: ${{ matrix.k8s-version}}
+          version: ${{ env.SCAFFOLDING_RELEASE_VERSION }}
+
+      - name: Install policy-controller-with-only-pod-resource
+        env:
+          GIT_HASH: ${{ github.sha }}
+          GIT_VERSION: ci
+          LDFLAGS: ""
+          POLICY_CONTROLLER_YAML: test/kustomize-custom-resource/policy-controller-e2e.yaml
+          KO_PREFIX: registry.local:5000/policy-controller
+          POLICY_CONTROLLER_ARCHS: linux/amd64
+        run: |
+          make ko-policy-controller
+          kustomize build test/kustomize-custom-resource | kubectl apply -f -
+
+          # Wait for the webhook to come up and become Ready
+          kubectl rollout status --timeout 5m --namespace cosign-system deployments/webhook
+          echo "TUF_ROOT_FILE=./root.json" >> $GITHUB_ENV
+
+      - name: Checkout TSA for testing.
+        uses: actions/checkout@11bd71901bbe5b1630ceea73d27597364c9af683 # v3.0.2
+        with:
+          repository: sigstore/timestamp-authority
+          path: ./src/github.com/sigstore/timestamp-authority
+
+      - name: Build timestamp-cli
+        working-directory: ./src/github.com/sigstore/timestamp-authority
+        run: |
+          go build -o ./timestamp-cli ./cmd/timestamp-cli
+
+      - name: Exercise our local TSA
+        working-directory: ./src/github.com/sigstore/timestamp-authority
+        run: |
+          TSA_URL=$(kubectl -n tsa-system get ksvc tsa -ojsonpath='{.status.url}')
+          echo "TSA_URL=$TSA_URL" >> $GITHUB_ENV
+
+          curl $TSA_URL/api/v1/timestamp/certchain > ts_chain.pem
+          echo "myblob" > myblob
+          if ! ./timestamp-cli --timestamp_server $TSA_URL timestamp --hash sha256 --artifact myblob --out response.tsr ; then
+            echo "failed to timestamp artifact"
+            exit -1
+          fi
+          if ! ./timestamp-cli verify --timestamp response.tsr --artifact "myblob" --certificate-chain ts_chain.pem ; then
+            echo "failed to verify timestamp"
+            exit -1
+          fi
+          if ! ./timestamp-cli inspect --timestamp response.tsr --format json ; then
+            echo "failed to inspect the timestamp"
+            exit -1
+          fi
+
+      - name: Run Custom resources Tests
```

Comment:
You can remove L141 to here, since this is the only thing we're really testing here. No need to test the TSA again.

Reply:
You are right, removed.